akka-stream/src/main/scala/akka/stream/ActorMaterializer.scala

```diff
@@ -284,6 +284,7 @@ final class ActorMaterializerSettings(
   val autoFusing: Boolean,
   val maxFixedBufferSize: Int) {
 
+  var syncProcessingLimit = 200 //defailt value
```

Comment:
Settings must be immutable, please add this limit in the same fashion as all the others. Binary compatibility is achieved by adding a secondary constructor that sets this limit to 200.